tensorflow/contrib/metrics/python/ops/metric_ops_test.py

```diff
@@ -1734,9 +1734,10 @@ def testPredictionsOutOfRange(self):
       predictions = constant_op.constant(
           [1, -1, 1, -1], shape=(1, 4), dtype=dtypes_lib.float32)
       labels = constant_op.constant([0, 1, 1, 0], shape=(1, 4))
-      _, update_op = metrics.streaming_auc(predictions, labels)
-      sess.run(variables.local_variables_initializer())
-      self.assertRaises(errors_impl.InvalidArgumentError, update_op.eval)
+      with self.assertRaisesRegexp(errors_impl.InvalidArgumentError,
+                                   r"predictions must be in \[0, 1\]"):
+        _, update_op = metrics.streaming_auc(predictions, labels)
+        # Error detected statically; no need to run the op.
```

Comment:
Unused variable 'update_op' [unused-variable]

Reply:
Changed `update_op` to `_`